platform/app/.webpack/webpack.pwa.js

```diff
@@ -79,6 +79,14 @@ module.exports = (env, argv) => {
         // Hoisted Yarn Workspace Modules
         path.resolve(__dirname, '../../../node_modules'),
         SRC_DIR,
+        path.resolve(
```

Comment:
I'm not really sure why this is needed when other modes and extensions don't require it. We should investigate why this is the case; it probably shouldn't be here.